scripts/README.md

```diff
@@ -1,8 +1,34 @@
-# Scripts
+- [Setup](#setup)
+- [Available\_Scripts](#available_scripts)
+  - [submit.sh](#submitsh)
+    - [Requirements](#requirements)
+    - [Usage](#usage)
+  - [rs.sh](#rssh)
+    - [Requirements](#requirements-1)
+    - [Usage](#usage-1)
+    - [Examples](#examples)
+  - [ti.sh](#tish)
+    - [Requirements](#requirements-2)
+    - [Usage](#usage-2)
+    - [Examples](#examples-1)
+  - [ucsd.sh](#ucsdsh)
+    - [Requirements](#requirements-3)
+    - [Before running the script](#before-running-the-script)
+    - [Usage](#usage-3)
+  - [setup/update-examples-snapshots.sh](#setupupdate-examples-snapshotssh)
+    - [Requirements](#requirements-4)
+    - [Usage](#usage-4)
+  - [setup/setup-reportstream.sh](#setupsetup-reportstreamsh)
+    - [Requirements](#requirements-5)
+    - [Usage](#usage-5)
+  - [lib/common.sh](#libcommonsh)
+  - [lib/submission-utils.sh](#libsubmission-utilssh)
+- [Resources](#resources)
 
 ## Setup
-
-Follow the instructions below to load the environments variables required for these scripts
+<details open>
```

Comment:
This **detail** tag is a collapsable section. I wanted to see how others feel about this so I am including it as a draft.

Reply:
Not sure about this. I think the collapsible section feature could be useful for longer readmes where we have sections that we want to hide/collapse by default, but I don't see why we'd need it here. Specially if it's expanded by default

Reply:
That's fair. I could probably make the `scripts` section collapsable but I think with the new generated ToC, it might not be needed.